sources/packages/backend/apps/api/src/testHelpers/fake-entities/program-year-fake.ts

```diff
@@ -1,12 +1,21 @@
 import { ProgramYear } from "@sims/sims-db";
 
-export function createFakeProgramYear(): ProgramYear {
+export function createFakeProgramYear(
+  programYearPrefix?: number,
+  active?: boolean,
```

Comment:
In general, we have not been passing parameters just for the sake of direct assignment, like `active`.
The idea is that we can invoke the `createFakeProgramYear` method and set the property there.
For instance, the `programYearPrefix` is a great example of a parameter that, when provided, is useful to set many other properties.